### ISSUE TITLE
[Feature Request] Round Headers

### ISSUE THREAD

**Issue by weebao** [natancabral/pdfkit-table#94]
It would be nice if we could change our headers' border radius.

From:
![image](https://github.com/user-attachments/assets/32da1b99-511b-4017-842e-73c52608063f)

To:
![image](https://github.com/user-attachments/assets/160f29bf-ee7e-49f1-a017-699216f85dc0)

**Comment by wilmanbarrios** [comment]
Hey 👋🏼 @weebao 

I'm trying to add the striped effect, but I'm unable to see the cell content, the rectangle gets in front of the text and I don't know how to put the text forward. 

Searching through the issues I found this one and, I can see that you have a table with the striped effect.

I wanted to ask, did you manage to achieve the striped effect or was it just to showcase the border radius?

**Comment by weebao** [comment]
Yeah I did. The way I did it was really manual though. I just used `map` and add `backgroundColor` for the odd rows to be gray lol. Couldn't find a better way 😅

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
